.circleci/config.yml

```diff
@@ -0,0 +1,79 @@
+version: 2
+
+jobs:
+  install-dependencies:
+    docker:
+      - image: circleci/node:lts
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - ui-dependencies-{{ checksum "ui/package-lock.json" }}
+            - ui-dependencies-
```

Comment:
npm ci is actually fast enough, thanks to the extremely high network speed of circleci, caching doesn't really buy us time

Reply:
⚠️  But you'll need caching if you revert to node 5.7 (npm ci does not exist on this version)